docs/dev/relay_add_op.rst

```diff
@@ -139,6 +139,50 @@ before producing the call node:
         tup = Tuple(list(args))
         return _make.concat(tup)
 
+Gradient Operators
+------------------
+
+Gradient operators are important for writing differentiable programs in

```

Comment:
Specifically you should say that gradient operators are necessary for the AD algorithm; the AD algorithm can differentiate first-class language constructs, but because operators are opaque, it needs an explicit differentiation rule for them.

Reply:
Nits:
* Do we have any documentation of Relay's AD? It would be good to link to that
* It would be good to say that the differentiation rule is opaque to Relay as well
* I think the sentence in which you describe operators as "opaque" should also contain the clause clarifying that "opaque" means that Relay cannot look into the implementation